### PR TITLE
Kolibri integration

### PR DESCRIPTION
**Pull request by swbsf**
Create a option for device_facts :
`kolibri: "True"`

### CODE REVIEW

File: roles/kolibri/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  become_user: ideascube
+  file: path=/home/ideascube/.kolibri state=directory
+  tags:
+    - custom
+
+- name: Install Kolibri in virtualenv
+  become: yes
+  become_user: ideascube
+  pip: 
+    name=kolibri
```
Comment (fheslouin) [inline roles/kolibri/tasks/main.yml]: I think we should set the kolibri version as well 
Something like: `version={{ ansible_local.device_list[generic_project_name].kolibri.version }}`

File: roles/set_custom_fact/files/device_list.fact [hunk 2]
```diff
@@ -2960,10 +2960,13 @@
 			}
 		},
 		"kb-bsf-scaleway": {
-			"kalite": {
-				"activated": "True",
-				"version": "0.16.9",
-				"language": ["fr"]
+			"kolibri": {
+				"activated": "True"
```
Comment (fheslouin) [inline roles/set_custom_fact/files/device_list.fact]: We can also set the `version`. What about the language ? Is there anything that we can configure from command line ?

File: roles/kolibri/files/kolibri.service [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Kolibri server
+
+[Service]
+User=ideascube
+Group=ideascube
+ExecStart=/home/ideascube/.kolibri/env/bin/python2 /home/ideascube/.kolibri/env/bin/kolibri start
```
Comment (fheslouin) [inline roles/kolibri/files/kolibri.service:7]: They are still using python2 ?

File: roles/kolibri/tasks/main.yml (hunk 1 above)
Comment (swbsf) [reply roles/kolibri/tasks/main.yml]: Good point, I'll update this.

File: roles/kolibri/files/kolibri.service (hunk 3 above)
Comment (swbsf) [reply roles/kolibri/files/kolibri.service:7]: From https://libraries.io/pypi/kolibri : both are acceptable. Do we have a preference for Python3?

Comment (fheslouin) [reply roles/kolibri/files/kolibri.service:7]: I would say yes ! Let's start directly with the most advanced version. Ideascube is python 3 anyway

Comment (bochecha) [reply roles/kolibri/files/kolibri.service:7]: We (the Ideascube developers) have a strong preference for Python 3. In fact, Ideascube does not work any more with Python 2, and we have no intention of "fixing" that.

As a result, Python 3 must be installed on the boxes anyway for Ideascube, so moving everything to Python 3 means (potentially) less things installed on the boxes.

---

> From https://libraries.io/pypi/kolibri : both are acceptable

This seems like a third-party website, and as such might not be entirely accurate.

The best way to know what versions of Python are supported is to look at what is tested by the developers.

In the case of Kolibri, you can find this file in their source tree: https://github.com/learningequality/kolibri/blob/develop/tox.ini

As you can see, they run the tests on Python 2.7, 3.4 and 3.5. That's perfect for us, Debian Jessie has 3.4 and Stretch (if we ever move to it) has 3.5.

Comment (swbsf) [reply roles/kolibri/files/kolibri.service:7]: After a few tests, not sure whose the culprit here ansible (py2) or pip (py2, pip3 not installed), but kolibri won't install in his virtualenv. So we'll stick with python2 for now as both are installed on our box and this is not about to change.

File: roles/kolibri/tasks/main.yml (hunk 1 above)
Comment (swbsf) [reply roles/kolibri/tasks/main.yml]: fixed.

File: roles/set_custom_fact/files/device_list.fact (hunk 2 above)
Comment (swbsf) [reply roles/set_custom_fact/files/device_list.fact]: Language, not yet. Soon. :)
For now, not much can be configured, appart from plugins. Which will come soon enough.

**Pull request merged**